apps/omg_child_chain/test/omg_child_chain/integration/fee_server_test.exs

```diff
@@ -163,31 +161,7 @@ defmodule OMG.ChildChain.Integration.FeeServerTest do
     end
   end
 
-  describe "fees ignored" do
-    setup do
-      Application.put_env(:omg_child_chain, :ignore_fees, true)
-      {:started, log, exit_fn} = start_fee_server()
-      assert log =~ "ignored"
-
-      on_exit(fn ->
-        exit_fn.()
-        Application.put_env(:omg_child_chain, :ignore_fees, false)
-      end)
-
-      :ok
-    end
-
-    test "fee server ignores file updates" do
-      assert {:ok, :ignore_fees} == FeeServer.transaction_fees()
-
-      assert refresh_fees() =~ "Updates have no effect"
-
-      assert {:ok, :ignore_fees} == FeeServer.transaction_fees()
-      assert server_alive?()
-    end
-  end
-
-  defp start_fee_server() do
+  defp start_fee_server do
```

Comment:
```suggestion
  defp start_fee_server() do
```

Reply:
What... how possible....

Reply:
My bad rebase 🙇

Reply:
No no, I mean credo should've complained. Looks like it's complaining but not exiting with bad status. Fixing in a separate PR.

Reply:
https://github.com/omisego/elixir-omg/pull/1303 (for reference)